docs/reference/errors-and-warnings/NU5127.md

```diff
@@ -0,0 +1,24 @@
+---
+title: NuGet Warning NU5127
+description: NU5127 Warning code
+author: zivkan
+ms.date: 09/20/2019
+ms.topic: reference
+ms.reviewer: karann
+f1_keywords: 
+  - NU5127
+---
+
+# NuGet Warning NU5127
+
+<pre>This package does not contain a lib/ or ref/ folder, and will therefore be treated as compatible for all frameworks. Since framework specific files were found under the build/ directory for net45, netstandard2.0, consider creating the following empty files to correctly narrow the compatibility of the package:
+-lib/net45/_._
+-lib/netstandard2.0/_._</pre>
+
+### Issue
+
+Projects using packages with `PackageReference` only use `lib/` and `ref/` assemblies to determine package compatibility. Therefore, a package without any `lib/` or `ref/` files will be considered compatible with all projects. However, if that package contains build files specific to one or more [Target Framework Monikers (TFMs)](../target-frameworks.md), a package consumer may expect the package to fail if none of the build files are used.
+
+### Solution
+
+As the warning message suggests, create an empty file named `_._` in the lib folder for the TFMs listed. This will allow NuGet to fail the restore for `PackageReference` projects when the project is incompatible with the package.
```

Comment:
I think you are referring to creating a file with name `_._` with `no file extension` as part of the solution.

Reply:
Sorry, I don't understand. Is your comment that I should make it more obvious that the nupkg should contain this file?

The file's extension is `_`, so I really don't understand your no file extension comment. A package without `lib/` files is advanced and probably was not created by packing a project (although with our MSBuild pack targets it is possible, at least in SDK style projects), so there may not be a project or solution to add a `_._` file to. How the customer adds the `_._` depends on how they created the package and I hope they don't need instructions and worst case can copy how their `build/` files were added to the package.